core/ir/opnd_api.h

```diff
@@ -1766,13 +1763,8 @@ typedef enum {
 #    define DR_REG_STOP_DR DR_REG_DR15    /**< End of debug register enum values */
 #    define DR_REG_START_CR DR_REG_CR0    /**< Start of control register enum values */
 #    define DR_REG_STOP_CR DR_REG_CR15    /**< End of control register enum values */
-/**
- * Last valid register enum value.  Note: DR_REG_INVALID is now smaller
- * than this value.
- */
-#    define DR_REG_LAST_VALID_ENUM DR_REG_K7
```

Comment:
Since these were documented and exported, we can't just delete them without breaking compatibility.  Easiest to just make new cross-arch aliases that are based on the new enum value to maintain compatibility.